ci/scripts/cpp_test.sh

```diff
@@ -80,6 +80,12 @@ pushd ${build_dir}
 if [ -z "${PYTHON}" ] && ! which python > /dev/null 2>&1; then
   export PYTHON="${PYTHON:-python3}"
 fi
+if [[ "${ARROW_ENABLE_THREADING}" == "OFF" ]]; then
+# if threading is disabled, some tests take longer to run
+# but we can get away with running more tests in parallel because we know they wont make loads of threads
```

Comment:
```suggestion
  # if threading is disabled, some tests take longer to run
  # but we can get away with running more tests in parallel because we know they won't make loads of threads
```